src/PrestaShopBundle/Entity/Repository/TabRepository.php

```diff
@@ -38,6 +38,15 @@ public function findByModule($moduleName)
     {
         return $this->findBy(['module' => $moduleName]);
     }
+
+    /**
+     * @param $idParent
+     * @return array
+     */
+    public function findByParentId($idParent)
```

Comment:
findByIdParent($idParent) should work without need to create it :)

Reply:
That was only for the IDE autocompletion, should I remove it?

Reply:
nope, you're right :)